training-schema.yml

```diff
@@ -0,0 +1,44 @@
+$id: https://ocr-d.github.io/schemas/v1/training-schema.json
+type: object
+required:
+  - engineName
+  - engineVersion
+  - groundTruthBag
+  - outputModelFormat
+properties:
+  engineName:
+    type: string
+    enum:
+      - ocropus
+      - kraken
+      - tesseract
+      - calamari
+  engineVersion:
+    type: string
+  engineArguments:
+    description: Command line arguments passed to the CLI training tool
+    type: array
+    default: []
+  groundTruthBag:
+    description: A bag of line ground truth adhering to https://ocr-d.github.io/gt-profile.json
+    type: string
+  groundTruthGlob:
+    description: Wildcard for matching only a subset of the ground truth files. Make sure to exclude extensions and end in '*'.
+    type: string
+    default: '*'
+  outputModelFormat:
+    description: The output format of the model. Note that individual engines only support a single one or a subset of formats.
+    enum:
+      - application/vnd.ocrd.pronn  # kraken < 1.0
+      - application/vnd.ocrd.clstm  # ocropy-lpred, clstm, kraken<1.0
+      - application/vnd.ocrd.coreml # kraken >= 1.0
+      - application/vnd.ocrd.pyrnn  # ocropy-rpred
+      - application/vnd.ocrd.tf+zip # calamari, zipped tensorflow data
```

Comment:
This enum does not contain a tesseract-specific format (at least not in the comments).

Reply:
Right. How about `application/vnd.ocrd.tesseract4`?

Reply:
Fine.